spec/factories/fee_types.rb

```diff
@@ -16,7 +16,7 @@
 FactoryGirl.define do
```

Comment:
this has been changed already (in develop,etc) to a simpler factory method that avoids any validation issues (based on code)